PurchasesCoreSwift/Logging/Strings/ConfigureStrings.swift

```diff
@@ -12,7 +12,7 @@ import Foundation
 @objc(RCConfigureStrings) public class ConfigureStrings: NSObject {
     @objc public var adsupport_not_imported: String { "AdSupport framework not imported. Attribution data incomplete." }
     @objc public var application_active: String { "applicationDidBecomeActive" }
-    @objc public var configuring_purchases_proxy_url_set: String {
+    @objc public class var configuring_purchases_proxy_url_set: String {
```

Comment:
why is this needed?

Reply:
It's not, I missed the usage of Strings.configure.
typically strings are held as a class var instead of an instance var, so this felt natural to me. I meant to go back and take a look at this, but I didn't. I'll revert this and go with `Strings.configure.`